trophic_levels-biomass_indicators.rmd

```diff
@@ -44,21 +44,21 @@ pData <- ifelse(dir == "C:",
 
 <!-- Document you work below. Try not to change  the headers too much. Data can be stored on NINA server. Since the book is rendered on the R Server this works fine, but note that directory paths are different on the server compared to you local machine. If it is not too big you may store under /data/ on this repository -->
 
-## Introduction
+## Introduction[?]
```

Comment:
The introduction is very nice. However, it would be great if yo could add a few paragraphs according to the chapter template (https://github.com/NINAnor/ecosystemCondition/blob/main/template.Rmd), headings "About the underlying data" to "Uncertainties"